mbus-api/src/message_bus/v0.rs

```diff
@@ -168,6 +202,113 @@ pub trait MessageBusTrait: Sized {
         let _ = request.request().await?;
         Ok(())
     }
+
+    /// Get nexuses with filter
+    #[tracing::instrument(level = "debug", err)]
+    async fn get_nexuses(filter: Filter) -> BusResult<Vec<Nexus>> {
+        let nexuses = GetNexuses {
+            filter,
+        }
+        .request()
+        .await?;
+        Ok(nexuses.0)
```

Comment:
I prefer to use .into_inner() rather then `.0` calls. into_inner() is more verbose but if we do this consistently a lot easier to manage in the future. This is the same approach taken by other crates.